browser/app/js/browser/Login.js

```diff
@@ -25,14 +25,35 @@ import web from "../web"
 import { Redirect } from "react-router-dom"
 
 export class Login extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      accessKey: "",
+      secretKey: ""
+    }
+  }
+
+  // Handle field changes
+  accessKeyChange(e) {
+    this.setState({
+      accessKey: e.target.value
+    })
+  }
+
+  secretKeyChange(e) {
+    this.setState({
+      secretKey: e.target.value
```

Comment:
Is this a good idea, how secure is State() does it use browser localStorage?? Am I assuming it's not transmitted on CORS requests?

Reply:
State is a javascript object lives within the scope of the component. It's not stored anywhere else.